protocol/linting.md

```diff
@@ -0,0 +1,16 @@
+# Proposed linting standards
+
+Linting is a best practice for developers in general and an essential aid for developers working together on open source projects. Here are kmcculloch's proposed linting standards for EDGI projects:
+
+1. All projects that constitute an application codebase (rather than a set of documents or stand-alone scripts) should adopt a **linting tool** and a **linting standard** that is appropriate for the language or framework.
```

Comment:
I think we almost want to have a line item (or all of this?  🤔) in our Project Guidelines... a la https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md (tho a little less intense)